README.md

```diff
@@ -250,7 +288,8 @@ Once this loads, connect to your running local cluster with:
 connect client 'localhost:1527';
 ```
 
-Set Spark shuffle partitions low since we don't have a lot of data; you can optionally view the members of the cluster as well:
+Set Spark shuffle partitions low since we don't have a lot of data; you can optionally view the members of the cluster
```

Comment:
"...partitions low..." - should be "...partitions to a lower number". Also mentioning the reason behind the relation between partition number and data size would be good.